drv/gimlet-seq-server/src/main.rs

```diff
@@ -138,7 +138,9 @@ fn main() -> ! {
             // All these moments will be lost in time, like tears in rain...
             // Time to die.
             loop {
-                hl::sleep_for(100);
+                if sys_recv_closed(&mut [], 1, TaskId::KERNEL).is_err() {
```

Comment:
This looks like it's the timer notification -- what's your intent here?

Reply:
We talked about this in chat -- the intent was to pick a notification that will never arrive. Unfortunately there are no such notifications, because _in theory_ any notification can be delivered using `sys_post`. So, this should probably just loop if it gets an errant notification, as we discussed.

Reply:
Yup, addressed Cliff's suggestions from our discussion in cbdbfc44018d9ce98c9e983d199966940e72cedd :)